source/MapPanel.cpp

```diff
@@ -894,16 +903,17 @@ void MapPanel::UpdateCache()
 	for(const auto &it : GameData::Systems())
 	{
 		const System *system = &it.second;
-		if(!system->IsValid() || !player.HasSeen(*system))
+		if(!system->IsValid() || system->Inaccessible() || !player.HasSeen(*system))
```

Comment:
Redundant check with the face that an inaccessible system will return an empty Links set.
```suggestion
		if(!system->IsValid() || !player.HasSeen(*system))
```